libs/remix-ui/run-tab/src/lib/actions/events.ts

```diff
@@ -103,12 +103,20 @@ export const setupEvents = (plugin: RunTab, dispatch: React.Dispatch<any>) => {
     if (plugin.name === 'remixd') {
       dispatch(setRemixDActivated(true))
     }
+
+    if (plugin.name === 'dapp-draft') {
```

Comment:
I m not sure why you need to do this.. when the user click on the icon to load dapp-draft, the plugin will be automatically activated.

Reply:
I'm not sure. I mean it's not a native plugin for now. Are you ok with the icon always show in deployed contract? I definitely don't need this if you're ok with that. @yann300

Reply:
If this requires the plugin to be activated I fear no one will see it. So, let's have this always displayed for now. As long as the feature is free to use, and doesn't imply any subscription, I think we'll be fine for that.
@LianaHus might want to check the UI here though.